kanidmd/src/lib/interval.rs

```diff
@@ -24,4 +29,51 @@ impl IntervalActor {
             }
         });
     }
+
+    pub fn start_online_backup(server: &'static QueryServerReadV1, cfg: &OnlineBackup) {
+        let outpath = cfg.path.to_owned();
+        let schedule = cfg.schedule.to_owned();
+        let versions = cfg.versions;
+
+        // TODO: add some checks arount the provided cron pattern .any() etc.
+        let cron_expr = match schedule.as_str().parse::<CronExpr>() {
+            Ok(ce) => ce,
+            Err(e) => {
+                info!(
+                    "Online backup schedule parse error {}. Pattern set to < 00 22 * * * >",
```

Comment:
If we fail to parse, we should return an error instead of proceeding with a default schedule.

Reply:
Agreed. Someting like `eprintln`? I think stopping the entire server would be too much. On the other hand, we should make sure this error is configuration error recognized and fixed. Because there might be large delayes before it is run. How to do this best?

Reply:
If you change the `pub fn start_online_backup(...) -> Result<(), ()>`, then you can do:

```
let cron_expr = schedule.as_str().parse::<CronExpr>()
    .map_err(|e| {
        error!("Online backup schedule parse error {}", e);
    })?;
```

That will cause the function to return, and then in kanidmd/src/lib/core/mod.rs you can do somethig like:

```
start_online_backup(...)?;
```

When you call it, which will cause create server core to shortcut and retuirn without completing the server start up process.